src/components/Tabs/Tabs.tsx

```diff
@@ -176,27 +122,19 @@ export const Tabs = ({
     }
   }
 
-  const componentClassName = clsx(styles['tabs'], className);
-
-  const headerClassName = clsx(
-    styles['tabs__header'],
-    scrollableLeft && styles['tabs--scrollable-left'],
-    scrollableRight && styles['tabs--scrollable-right'],
-  );
-
   const activeTabPanel = React.cloneElement(tabs[activeIndexState], {
     id: activeTabPanelId,
     'aria-labelledby': tabIds[activeIndexState],
   });
 
   return (
-    <div className={componentClassName} {...other}>
-      <div
-        className={headerClassName}
-        onScroll={(e) => handleTabsScroll(e.target as HTMLDivElement)}
-        ref={headerRef}
-      >
-        <ul className={styles['tabs__list']} role="tablist">
+    <div className={clsx(styles['tabs'], className)} {...other}>
+      <div className={styles['tabs__header']}>
+        <ul
+          aria-labelledby={ariaLabelledBy}
```

Comment:
Moved the `aria-labelledby` to the element with the tablist role, because that's how https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html does it. It also makes sense that its the tablist that would be labelled, not some random div.

I will note, however, that the WAI-ARIA tabs example is just that - an example. It's not a guideline or requirement. But I like following it when possible.